.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Actual Behavior
```

Comment:
I have to disagree Sean. I think it is more natural to say, here is the issue I am having. Most issues I see start with a paste of the error.